### ISSUE TITLE
LUA errors since updating to 90205.6-release

### ISSUE THREAD

**Issue by jezaldanluthar** [casualshammy/NameplateAuras#155]
Getting 100s of lua errors since updateing to the newest release. Happens after every reload:

```
Message: AceLocale-3.0: NameplateAuras: Missing entry for 'options:general:hide-player-blizz-frame'
Time: Wed Jul 20 22:24:45 2022
Count: 1
Stack: AceLocale-3.0: NameplateAuras: Missing entry for 'options:general:hide-player-blizz-frame'
[string "@Interface\AddOns\AddOnSkins\Libs\Ace3\AceLocale-3.0\AceLocale-3.0.lua"]:31: in function <...AddOnSkins\Libs\Ace3\AceLocale-3.0\AceLocale-3.0.lua:29>
[string "@Interface\AddOns\NameplateAuras\src\options-dialog.lua"]:122: in function <...terface\AddOns\NameplateAuras\src\options-dialog.lua:95>
[string "@Interface\AddOns\NameplateAuras\src\options-dialog.lua"]:4515: in function <...terface\AddOns\NameplateAuras\src\options-dialog.lua:4419>
[string "@Interface\AddOns\NameplateAuras\src\options-dialog.lua"]:4635: in function `ShowGUI'
[string "@Interface\AddOns\NameplateAuras\src\NameplateAuras.lua"]:287: in function `?'
[string "@Interface\FrameXML\ChatFrame.lua"]:5188: in function `ChatEdit_ParseText'
[string "@Interface\FrameXML\ChatFrame.lua"]:4852: in function `ChatEdit_SendText'
[string "@Interface\FrameXML\ChatFrame.lua"]:4888: in function <Interface\FrameXML\ChatFrame.lua:4884>
[string "=[C]"]: in function `ChatEdit_OnEnterPressed'
[string "*:OnEnterPressed"]:1: in function <[string "*:OnEnterPressed"]:1>

Locals: <none>
```

**Screenshots**

https://imgur.com/0tCBZa2
![image](https://user-images.githubusercontent.com/47871184/180075042-b83dc99b-21c0-45ae-b453-adf3d08e43db.png)

![image](https://user-images.githubusercontent.com/47871184/180075393-7d3b1251-ae56-4ebd-a72b-41ffd9dc8ff1.png)

![image](https://user-images.githubusercontent.com/47871184/180091219-0141b039-b761-496a-87af-f5bc0f768416.png)

**Comment by TimWinnicheck** [comment]
second that

**Comment by casualshammy** [comment]
Thanks for reporting! I hope this issue is addressed in 90205.7-release

**Comment by jezaldanluthar** [comment]
> Thanks for reporting! I hope this issue is addressed in 90205.7-release

yep fixed! thanks
